### PR TITLE
Add Load from string for settings manager + misc changes

### PR DESCRIPTION
**Pull request by BenCat07**
- Add a Missing Spellbook (which is actually an orb) to spellbook list

- Forward declare the queue_time Timer to allow Modules to stop catbots from exiting after 20 minutes of main menu idle

- Add a Way to load a config from a string (mostly for Modules)

- Add a bumper cart hack (thanks to UNKN0WN)

- Fix crashes and buggy text when drawing too much text (e.g. entity esp)

### CODE REVIEW

File: src/settings/SettingsIO.cpp [hunk 1]
```diff
@@ -132,6 +132,33 @@ bool settings::SettingsReader::loadFrom(std::string path)
     return true;
 }
 
+bool settings::SettingsReader::loadFromString(std::string stream)
+{
+    settings::SettingsReader loader{ settings::Manager::instance() };
+    if (stream == "")
+    {
+        logging::Info("cat_load: Can't access file!");
```
Comment (TotallyNotElite) [inline src/settings/SettingsIO.cpp]: What file?

File: src/settings/SettingsIO.cpp [hunk 2]
```diff
@@ -132,6 +132,33 @@ bool settings::SettingsReader::loadFrom(std::string path)
...
+        g_ICvar->ConsoleColorPrintf(MENU_COLOR, "CAT: cat_load: File doesn't exist / can't open file!\n");
```
Comment (TotallyNotElite) [inline src/settings/SettingsIO.cpp]: Not a file

File: src/settings/SettingsIO.cpp (hunk 1 above)
Comment (BenCat07) [reply src/settings/SettingsIO.cpp]: resolved

File: src/settings/SettingsIO.cpp (hunk 2 above)
Comment (BenCat07) [reply src/settings/SettingsIO.cpp]: resolved

**Pull request merged**